api/app/signals/apps/questionnaires/services/feedback_request.py

```diff
@@ -0,0 +1,176 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+"""
+Support for Klanttevredenheidsonderzoek flow.
+"""
+from datetime import timedelta
+
+from django.db import transaction
+from django.utils import timezone
+
+from signals.apps.feedback.models import Feedback, StandardAnswer
+from signals.apps.feedback.utils import get_fe_application_location
+from signals.apps.questionnaires.app_settings import FEEDBACK_REQUEST_DAYS_OPEN
+from signals.apps.questionnaires.exceptions import SessionNotFrozen, WrongFlow, WrongState
+from signals.apps.questionnaires.models import (
+    Choice,
+    Edge,
+    Question,
+    QuestionGraph,
+    Questionnaire,
+    Session
+)
+from signals.apps.signals import workflow
+from signals.apps.signals.models import Signal
+
+
+def create_kto_graph():
```

Comment:
Great function to replicate the current feedback form as a questionnaire. However we still need to think on what to do with these kind of questionnaires. Because they are generated automatically they can or cannot be edited by an admin? Something we need to fix but not in this PR perse.

Perhaps we can also move this function to its own file as it is more of a utils function. So `utils.py` or something like that would be a great place.